### ISSUE TITLE
[Adaptive Cards-InputsWithValidations.json]: Tooltip is not defined for "Next" and "Previous" year buttons on calendar picker under 'InputsWithValidations.json' card.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9239]
### Target Platforms

UWP

### SDK Version

1.0.3.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Visualizer
OS Version: 25H2 (OS Build 26200.5670) 
App Version: 1.0.3.0

**Repro steps:**

1. Open the Adaptive Card Visualizer application.
2. Navigate to 'InputsWithValidations.json' button by using Tab key and activate it.
3. 'InputsWithValidations.json' adaptive card will be appeared on right sight of the pane.
4. Navigate to "Calendar" picker button and activate it.
5. Observe the issue on Next and Previous buttons.

**Actual Result:**
Tooltip is not defined for "Next" and "Previous" year buttons on calendar picker under 'InputsWithValidations.json' card.

**Note**
1. Same issue replicates for the "Next" and "Previous" year buttons on calendar picker under 'ActivityUpdate.json' card.
2. Same issue is also repro with "Accept" and "Dismiss" buttons on 'InputsWithValidations.json' card under Time picker button.

**Expected Result:**
Tooltip such as, "Next year button" for Next button and "Previous year button" for Previous button should be defined.

**User Impact:**
Users who rely on AT will face difficulty if the button controls doesn't have tool-tip defined, users may not know what the purpose of the element if no tool-tip is present.

Have feedback to share on Bugs? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Image](https://github.com/user-attachments/assets/5da4c951-8d8d-4446-bbfc-9fda45f86986)

![Image](https://github.com/user-attachments/assets/c7cc43a3-1176-44bd-9343-f48ad3e489de)

https://github.com/user-attachments/assets/e2adaac2-60f1-40d3-b946-5529c378a950

### Card JSON

```JSON
N/A
```

**Comment by Rishikesh6521** [comment]
Original Bug ID: #7683
